### DISCUSSION TITLE
Place Window in Middle of Dual-Display Monitor

### DISCUSSION THREAD

**Discussion by ArneTR** [rxhanson/Rectangle#633] [Ideas]
Is it possible to add an option to place a window, while retaining it's current size, in the center of a dual-display monitor?

I currently have a curved display with two 2560x1440 displays. They have no border in between them, so it can be used as one  very wide monitor.

In focused mode I often only use one main window and position it in the center of my view. My current workflow with rectangle is to maximize the window on the right display (which makes it the full size of one display) and then use the mouse to move it half of it's width to the left.

To my knowledge moving the window half of it's width to one side between displays is not yet possible. 

Therefore I wanted to propose this feature request.

**Comment by rxhanson** [comment]
I don't think I would plan on adding this in myself, but I see no reason to reject a good PR for it. It could easily be done like the "specified" command, which is only available via Terminal command and not in the UI.

It should be possible to configure it using the custom shortcut builder in my paid superset of Rectangle, Hookshot (https://hookshot.app), although that might be a little less ideal because you would have to manually figure out where the best custom docking position lies.

**Reply by shalmi** [reply]
I am also looking for multi monitor support.
I may actually work on this if you are actually willing to pull in a pull request for it.

**Reply by rxhanson** [reply]
Go for it. 

Here's the PR that was for the "specified" command to use as a reference: https://github.com/rxhanson/Rectangle/pull/478/files

Here's how it is configured via terminal commands: https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#add-an-extra-centering-command-with-custom-size
